app_dart/lib/src/request_handlers/update_task_status.dart

```diff
@@ -160,9 +160,9 @@ class UpdateTaskStatus extends ApiRequestHandler<UpdateTaskStatusResponse> {
     final String builderName = requestData[builderNameParam] as String;
     final Query<Task> query = datastore.db.query<Task>(ancestorKey: commitKey);
     final List<Task> initialTasks = await query.run().toList();
-    // Expected initialTasks will return one CocoonAgentTask and one LuciTask.
```

Comment:
This was out of date. Currently it pulls all tasks for a commit and scans them for the one with the builder set.

Originally, it used the task name.